news/20210720144841.misc

```diff
@@ -0,0 +1 @@
+Get `forced_reset_timeout` from mbed-os/targets/targets.json and add it to generated mbed_config.cmake as `MBED_GREENTEA_TEST_RESET_TIMEOUT` for use by CTest and mbedhtrun.
```

Comment:
This should be a `.feature`, not a `.misc`.

See guidance in our contributing guide: https://github.com/ARMmbed/mbed-tools/blob/master/CONTRIBUTING.md#news-files

Reply:
Done. As for the date in the filename, the example in the guide is precise to year, month and date only, not hours, minutes and seconds like in many existing PRs, so I shortened it to date too.

Reply:
If we just use the date, and two PRs are raised on the same day that happen to add, for instance, a ".bugfix" newsfile, we'll get merge conflicts. That's why we usually use a more precise date in the filename, or even better, an issue number. The guide needs to be updated.

Reply:
Fair enough. Probably precision to minute is more than enough - I often have to make up seconds 😅  because some other PRs have seconds.

Reply:
I just use the date command e.g `nvim news/`date -u "+%Y%m%d%H%M%S"`.bugfix`